openvdb/openvdb/points/PointRasterizeSDF.h

```diff
@@ -301,17 +302,34 @@ struct EllipsoidSettings
     using BaseT::radius;
     using BaseT::radiusScale;
 
-    /// @param rotation  the attribute containing each points orthogonal
-    ///   rotation matrix.
-    /// @details  This attribute must exist and represents the rotation of
-    ///   each points ellipse. Must be a Mat3s type.
-    std::string rotation = "rotation";
+    /// @param xform  the attribute containing each points transformation
+    /// @details  This attribute must exist and represents the xform of
+    ///   each points ellipse. Must be a Mat3s (float) or Quatf type. Note
+    ///   that if it is a matrix type, any scale represented by the matrix
+    ///   is combined with the radius and radiusScale values (and is
+    ///   interpreted in world space).
+    std::string xform = "xform";
 
     /// @param pws  An optional attribute which represents the world space
     ///   position of a point.
     /// @details  This can be useful to override the position of a point in
     ///   index space. If it exists, it must be a Vec3d type.
     std::string pws = "";
+
+    /// Old style "xform" attribute which has since been renamed and ONLY
+    /// supported rotation. If set, will be prioritised and infered to only
+    /// hold a rotation. This will be removed and should not be used.
```

Comment:
This comment is a bit confusing, Maybe 'Before 'xform' support was added, only rotation attributes were supported. This has been superceded and should no longer be used. If set, will be prioritised and newer 'xform' will be ignored for backwards compatibility.'